apps/prairielearn/src/lib/db-types.ts

```diff
@@ -267,3 +267,11 @@ export const StripeCheckoutSessionSchema = z.object({
   subject_user_id: IdSchema.nullable(),
 });
 export type StripeCheckoutSession = z.infer<typeof StripeCheckoutSessionSchema>;
+
+export const GroupRoleSchema = z.object({
+  id: z.string(),
+  role_name: z.string(),
+  minimum: z.number().nullable(),
+  maximum: z.number().nullable(),
+});
```

Comment:
We only used this schema in tests and therefore only included fields relevant to the tests, but adding all columns from the DB table makes sense so that this schema can be used not just for specific testing cases, but anywhere.